exampleSite/content/docs/guide/shortcodes/icon.fa.md

```
@@ -20,28 +20,35 @@ enableInlineShortcodes: true
-پرونده `data/icons.yaml` را ایجاد کنید، سپس آیکون‌های SVG خود را در فرمت زیر اضافه کنید:
+فایل `data/icons.yaml` را ایجاد کنید، سپس آیکون‌های SVG خود را با فرمت زیر اضافه کنید:
 
 ```yaml {filename="data/icons.yaml"}
-your-icon: <svg>your icon svg content</svg>
+your-icon: <svg>محتوای SVG آیکون شما</svg>
 ```
 
-سپس می‌توان آن را در کد کوتاه مانند زیر استفاده کرد:
+سپس می‌توانید از آن در شورت‌کد به این صورت استفاده کنید:
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)